tw-tkms-starter/build.gradle

```diff
@@ -75,8 +77,8 @@ if (System.getenv("IN_CI") != "true") {
 dockerCompose {
     useComposeFiles = ["src/test/resources/docker-compose.yml"]
     projectName = "tw-tkms-starter"
-    // Create some flakiness on slower comps
-    // waitForTcpPorts = false
+    // May create some flakiness on slower comps
+    waitForTcpPorts = false
```

Comment:
I would generally prefer this one. You don't stop containers, so this slowdown should not be very noticeable. 
Flakiness and noice on the other hand will be noticeable and harder to track, as most of our docker setups wait for all containers to be available.

Plus, the wait gives you confidence that your docker compose is correct, before proceeding with the tests, which again, help debug issues.

Reply:
The `true` seems to still add couple of seconds of delay for some weird reason.